tests/ha/clvmd_lvmlockd.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Configure clvmd or lvmlockd
+# Maintainer: Loic Devulder <ldevulder@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use version_utils qw(is_sle sle_version_at_least);
+use utils 'zypper_call';
+use testapi;
+use lockapi;
+use hacluster;
+
+sub run {
+    my $lvm_conf = '/etc/lvm/lvm.conf';
+    my $lock_mgr = undef;
+
+    # lvmlockd is only available in SLE15+
+    if (get_var("USE_LVMLOCKD")) {
+        die 'lvmlockd can be only used on SLE15+' unless (is_sle && sle_version_at_least('15'));
+        $lock_mgr = 'lvmlockd';
+    }
+    else {
+        $lock_mgr = 'clvm';
+    }
+
+    # Wait until clvmd/lvmlockd test is initialized
+    barrier_wait("LOCK_INIT_$cluster_name");
+
+    # DLM process/resource needs to be started
+    die 'dlm is not running' unless check_rsc "dlm";
+    ensure_process_running 'dlm_controld';
+
+    # Test if packages are installed
```

Comment:
Comment is not applicable for case with lvmlockd because you not just checking that something is installed but trying to install it